resources/metadata.json

```diff
@@ -7,5 +7,5 @@
     "gettext-domain": "rounded-window-corners@fxgn",
     "settings-schema": "org.gnome.shell.extensions.rounded-window-corners",
     "session-modes": ["user", "unlock-dialog"],
-    "shell-version": ["45", "46"]
```

Comment:
The readme should also be changed to say "Supports GNOME 46+" instead of 45+

Reply:
Fixed in e92ccc3555d99e335da4b74fdb08355821458861